docs/docsite/rst/dev_guide/developing_modules_general.rst

```diff
@@ -48,6 +48,133 @@ Creating a development environment (platform-agnostic steps)
    ``$ . venv/bin/activate && . hacking/env-setup``
 
 
+Creating an info module
```

Comment:
In the text, you sometimes write info and sometimes facts module, which is I think somewhat confusing, especially since they are related, but different things.
```suggestion
Creating an info or a facts module
```